### DISCUSSION TITLE
How can I set a raw string as an arbitrary type?

### DISCUSSION THREAD

**Discussion by allanbbs** [nlohmann/json#4348] [Q&A] [answered]
If i'm 100% sure that a string is in a valid format, how can I set the string as its internal JSON value? E.g

x  = "32"
json.set("age",x) -> in this instance age would have the int 32 as a value, instead of the string. If x = "\\"32\\"" , it would then be a string. Is this possible, or is there a workaround for that?

**Comment by nlohmann** [comment, marked as answer]
I am not entirely sure what you want to accomplish, but you can set a string with

```cpp
json x = "32";
```
